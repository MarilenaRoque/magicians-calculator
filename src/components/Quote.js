import React from 'react';

export default function Quote() {
  const quoteStyle = {
    display: 'flex',
    displayDirection: 'column',
    alignItems: 'center',
    justifyContent: 'center',
    height: '70vh',
  };
  return (
    <div style={quoteStyle}>
      <h3 style={{ fontSize: '30px' }}>What did the triangle say to the circle? “You’re pointless.”</h3>
    </div>
  );
}
