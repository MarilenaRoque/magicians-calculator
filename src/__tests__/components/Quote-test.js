import renderer from "react-test-renderer";
import Quote from "../../components/Quote";


describe('Quote page Component', () => {

  const tree = renderer.create(<Quote></Quote>).toJSON();

  it("renders Quote page correctly", () => {
    expect(tree).toMatchSnapshot();
  });
  
})