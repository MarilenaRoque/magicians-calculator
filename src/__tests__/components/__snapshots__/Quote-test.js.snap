// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page Component renders Home page correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
      "justifyContent": "center",
      "margin": "auto",
      "width": "70%",
    }
  }
>
  <h2
    style={
      Object {
        "fontSize": "25px",
      }
    }
  >
    Welcome to Math Magicians!
  </h2>
  <p
    style={
      Object {
        "fontSize": "25px",
      }
    }
  >
    Math magicians is the tool that you needed to solve your mathematical problems in an easy, faster and fun way. We want to help you and guide you through the Magic Math Land and make you explore the best mathematical experiences.
  </p>
  <p
    style={
      Object {
        "fontSize": "25px",
      }
    }
  >
    Okay...I admit that was corny, but we all know math isn’t always the most exciting subject to teach. That means you have to find strategies to make lessons fun, like gamification in the classroom, math puzzles or -- in this case -- math jokes that will lighten the mood and brighten the vibe in your classroom.
  </p>
</div>
`;

exports[`Quote page Component renders Quote page correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "displayDirection": "column",
      "height": "70vh",
      "justifyContent": "center",
    }
  }
>
  <h3
    style={
      Object {
        "fontSize": "30px",
      }
    }
  >
    What did the triangle say to the circle? “You’re pointless.”
  </h3>
</div>
`;
